optin/reporting/package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "lint": "eslint .",
     "lint:fix": "eslint --fix .",
-    "test": "mocha --full-trace --recursive ./test",
+    "test": "mocha --full-trace --recursive ./test --exit",
```

Comment:
Why did we need to add the `--exit` option? Ideally we shouldn't need it.
Pls see this => https://stackoverflow.com/a/52143003/3190131

Reply:
hmm, without `--exit` the terminal just hanging there.
I'm not sure what caused the problem, I will double-check on that.